.github/workflows/release.yml

```diff
@@ -63,3 +65,14 @@ jobs:
           KEYSTORE_ENTRY_PASSWORD: ${{ secrets.KEYSTORE_ENTRY_PASSWORD }}
         run: |
           npx semantic-release
+          
+      - name: Find APK file
+        id: find_apk
```

Comment:
Why is the step given an ID?

Reply:
This is a placeholder from GitHub Copliot inline code suggestions, can be remove.